### DISCUSSION TITLE
Webview2 cant load page

### DISCUSSION THREAD

**Discussion by WaseemMassas** [MicrosoftEdge/WebView2Feedback#3931] [Q&A]
I created a windows form a project that had webview2 inside to navigate into the web URL, All things work fine when I run the project inside the visual studio, but when I create setup and install the project (at the same computer), the webview2 cant loads the URL.
Any help!

**Comment by champnic** [comment]
Hey @WaseemMassas - Are you getting an error if you debug the installed project? Do you have the WebView2Loader.dll getting installed along with your app?
